doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -1,137 +1,58 @@
-:orphan:
-
 .. _ncs_release_notes_changelog:
 
-Changelog for |NCS| v1.9.99
-###########################
+Changelog for |NCS| v1.9.2-dev1
+###############################
 
 .. contents::
    :local:
    :depth: 2
 
 The most relevant changes that are present on the main branch of the |NCS|, as compared to the latest official release, are tracked in this file.
 
-.. note::
-   This file is a work in progress and might not cover all relevant changes.
-
 .. HOWTO
 
    When adding a new PR, decide whether it needs an entry in the changelog.
    If it does, update this page.
    Add the sections you need, as only a handful of sections is kept when the changelog is cleaned.
    "Protocols" section serves as a highlight section for all protocol-related changes, including those made to samples, libraries, and so on.
 
-Highlights
-**********
-
-There are no entries for this section yet.
-
 Known issues
 ************
 
 Known issues are only tracked for the latest official release.
-See `known issues for nRF Connect SDK v1.9.0`_ for the list of issues valid for the latest release.
+See `known issues for nRF Connect SDK v1.9.2`_ for the list of issues valid for the latest release.
 
 Changelog
 *********
 
 The following sections provide detailed lists of changes by component.
 
-Application development
-=======================
-
-|no_changes_yet_note|
-
 Protocols
 =========
 
 This section provides detailed lists of changes by :ref:`protocol <protocols>`.
-See `Samples`_ for lists of changes for the protocol-related samples.
-
-Matter
-------
-
-|no_changes_yet_note|
-
-Applications
-============
-
-This section provides detailed lists of changes by :ref:`application <applications>`.
-
-nRF9160: Asset Tracker v2
--------------------------
-
-|no_changes_yet_note|
-
-Samples
-=======
-
-This section provides detailed lists of changes by :ref:`sample <sample>`, including protocol-related samples.
-For lists of protocol-specific changes, see `Protocols`_.
-
-Bluetooth samples
------------------
-
-|no_changes_yet_note|
-
-Drivers
-=======
-
-This section provides detailed lists of changes by :ref:`driver <drivers>`.
-
-|no_changes_yet_note|
-
-Libraries
-=========
-
-This section provides detailed lists of changes by :ref:`library <libraries>`.
-
-Bluetooth libraries and services
---------------------------------
-
-|no_changes_yet_note|
 
-sdk-nrfxlib
------------
+Bluetooth LE
+------------
 
-See the changelog for each library in the :doc:`nfxlib documentation <nrfxlib:README>` for additional information.
+* Updated:
 
-Scripts
-=======
+  * Removed calls to ``__WFE()`` for the nRF5340 SoC in MPSL, to be in line with the Errata for the Anomaly 160.
 
-This section provides detailed lists of changes by :ref:`script <scripts>`.
+* Fixed:
 
-Unity
------
+  * Fixed a rare issue that could cause an assert when requesting timeslots with type ``MPSL_TIMESLOT_REQ_TYPE_EARLIEST``.
```

Comment:
```suggestion
  * A rare issue that could cause an assert when requesting timeslots with type ``MPSL_TIMESLOT_REQ_TYPE_EARLIEST``.
```

Reply:
This should be actually a single line, without the "Fixed:" before it. Thanks for spotting this.